projects/packages/my-jetpack/src/products/class-videopress.php

```diff
@@ -156,4 +157,14 @@ public static function get_manage_url() {
 			return admin_url( 'admin.php?page=jetpack#/settings?term=videopress' );
 		}
 	}
+
+	/**
+	 * Checks whether the current plan (or purchases) of the site already supports the product
+	 *
+	 * @return boolean
+	 */
+	public static function has_required_plan() {
+		// using second argument `true` to force fetching from wpcom
+		return Current_Plan::supports( 'videopress-1tb-storage', true );
```

Comment:
This works for Jetpack sites. Since only JP sites have My Jetpack, this is fine, but if/when MJ is added to other site types, this can become a problem again.